source/common/runtime/runtime_features.cc

```diff
@@ -67,6 +67,8 @@ FALSE_RUNTIME_GUARD(envoy_reloadable_features_allow_multiple_dns_addresses);
 FALSE_RUNTIME_GUARD(envoy_reloadable_features_unified_mux);
 // TODO(alyssar) flip false once issue complete.
 FALSE_RUNTIME_GUARD(envoy_restart_features_no_runtime_singleton);
+// TODO(birenroy) flip after Google tests
```

Comment:
Add the dependency on the `envoy_reloadable_features_http2_new_codec_wrapper` flag.

Reply:
Done, thanks.

Reply:
We should ask others outside of Google to test this as well. Maybe say 'after burn-in interval'

Reply:
Done, thanks.